modules/helm/cmd.go

```diff
@@ -19,34 +18,34 @@ func getCommonArgs(options *Options, args ...string) []string {
 	if options.KubectlOptions != nil && options.KubectlOptions.ConfigPath != "" {
 		args = append(args, "--kubeconfig", options.KubectlOptions.ConfigPath)
 	}
-	if options.KubectlOptions != nil && options.KubectlOptions.Namespace != "" {
-		args = append(args, "--namespace", options.KubectlOptions.Namespace)
-	}
 	if options.HomePath != "" {
 		args = append(args, "--home", options.HomePath)
 	}
 	return args
 }
 
-// RunHelmCommandAndGetOutputE runs helm with the given arguments and options and returns stdout/stderr.
-func RunHelmCommandAndGetOutputE(t *testing.T, options *Options, cmd string, additionalArgs ...string) (string, error) {
-	args := []string{cmd}
-	args = getCommonArgs(options, args...)
-
+// getValuesArgsE computes the args to pass in for setting values
+func getValuesArgsE(t *testing.T, options *Options, args ...string) ([]string, error) {
 	args = append(args, formatSetValuesAsArgs(options.SetValues)...)
 
 	valuesFilesArgs, err := formatValuesFilesAsArgsE(t, options.ValuesFiles)
 	if err != nil {
-		return "", errors.WithStackTrace(err)
+		return args, errors.WithStackTrace(err)
 	}
 	args = append(args, valuesFilesArgs...)
 
 	setFilesArgs, err := formatSetFilesAsArgsE(t, options.SetFiles)
 	if err != nil {
-		return "", errors.WithStackTrace(err)
+		return args, errors.WithStackTrace(err)
 	}
 	args = append(args, setFilesArgs...)
+	return args, nil
+}
 
+// RunHelmCommandAndGetOutputE runs helm with the given arguments and options and returns stdout/stderr.
+func RunHelmCommandAndGetOutputE(t *testing.T, options *Options, cmd string, additionalArgs ...string) (string, error) {
+	args := []string{cmd}
+	args = getCommonArgs(options, args...)
```

Comment:
This refactor was necessary because only `install` and `upgrade` take the `--set`, `--set-file`, and `-f` options.